R/vt_template.R

```diff
@@ -0,0 +1,50 @@
+#' templating function
+#'
+#' Access valtools templates and create modified version based on the input data
+#'
+#' @param path output path to save rendered template to
+#' @param template template to use from valtools/inst/templates
+#' @param data named list or environment with variables that will be used during rendering
+#'
+#' @importFrom whisker whisker.render
+#' @importFrom rlang abort
+#'
+#' @noRd
+#'
+render_template <- function(template, output = template, data = list()){
+
+  template_path <- file.path(
+    system.file("templates", package = "valtools"),
+    template)
+
+  if(!file.exists(template_path)){
+    abort(paste0(c("Error during creation of template `",template,"`. Error: ",
+                   e, sep = "\n")),
```

Comment:
Can this message be made more specific? I don't think `e` exists in this context.

Reply:
:facepalm: yes, it should be a clearer "template does not exist". Of course e doesn't exist